ble/GattServer.h

```diff
@@ -259,12 +275,16 @@ class GattServer {
      * @Note: It is also possible to set up a callback into a member function of
      * some object.
      */
-    void onDataWritten(void (*callback)(const GattWriteCallbackParams *eventDataP)) {dataWrittenCallChain.add(callback);}
+    void onDataWritten(const DataWrittenCallback_t& callback) {dataWrittenCallChain.add(callback);}
     template <typename T>
     void onDataWritten(T *objPtr, void (T::*memberPtr)(const GattWriteCallbackParams *context)) {
         dataWrittenCallChain.add(objPtr, memberPtr);
     }
 
+    DataWrittenCallbackChain_t& onDataWritten() {
```

Comment:
please add a documentation header. it might be good to add a code-example showing its use